packages/launcher/src/main/index.js

```diff
@@ -47,6 +47,7 @@ const newWindow = (params: Object = {}) => {
     width: params.width || 800,
     height: params.height || 600,
     show: false,
+    titleBarStyle: 'hidden',
```

Comment:
Has this been tested on Window and Linux?

Reply:
not really, but as I read, this will only make changes to mac no?

Reply:
I don't have a way to test in linux or windows? do you know how I can do it?

Reply:
If it only affects macOS that's fine, but better make sure the UX is consistent on Linux and Windows. You can use VMs for example with VirtualBox to check on these systems.

Reply:
K I'll take a look

Reply:
Works well in Windows too @PaulLeCam no issues.